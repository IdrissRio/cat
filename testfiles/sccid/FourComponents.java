// PackageName: org.extendj.callgraph.FourComponents
// MethodName: foo

package org.extendj.callgraph;

public class FourComponents {
  void foo() { bar(); }

  void bar() { baz(); }

  void baz() { zzz(); }
  void zzz() {}
}