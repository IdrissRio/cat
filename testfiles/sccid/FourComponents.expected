org.extendj.callgraph.FourComponents::foo()->0
org.extendj.callgraph.FourComponents::bar()->1
org.extendj.callgraph.FourComponents::baz()->2
org.extendj.callgraph.FourComponents::zzz()->3

